docs/blog/introduction-0.13.0/index.qmd

```diff
@@ -0,0 +1,270 @@
+
+```{python}
+gt_tbl = (
+    gt_tbl
+    .tab_style(
+        style=[style.fill(color="Linen"), style.text(weight="bold")],
+        locations=loc.stub()
+    )
+    .tab_style(
+        style=style.fill(color="LightCyan"),
+        locations=[
+            loc.body(rows=pl.col("half_life").is_null()),
+            loc.stub(rows=pl.col("half_life").is_null())
+        ]
+    )
+ )
+
+gt_tbl
+```
+
+We've dressed up the stub so that it is that much more prominent. And that linen-colored stub goes so well with the light-cyan rows, representative of carbon-12 and carbon-13!
+
+### Using custom style rules with the new `style.css()`
+
+Aside from decking out the `loc` module with all manner of location methods, we've added a little something to the `style` module: `style.css()`! What's it for? It lets you supply style declarations to its single `rule=` argument. You have to study up on CSS style declarations (and which of those make sense in a table context) but if you know of certain styles that aren't covered in either of `style.fill()`, `style.text()`, or `style.borders()` then `style.css()` will be of major help here.
+
+As an example, I might want to indent some text in one or more table cells. You can't really do that with the `style.text()` method since it doesn't have an `indent=` argument. So, in Great Tables `0.13.0` you can do this:
+
+```{python}
+gt_tbl = (
+    gt_tbl
+    .tab_style(
+        style=style.css(rule="text-indent: 4px;"),
+        locations=loc.stub(rows=pl.col("half_life").is_null())
+    )
+)
+
+gt_tbl
+```
+
+We targeted the cells in the stub that corresponded to the stable isotopes (carbon-12 and -13) with a Polars expression (same one as in the previous code cell) and now we have a 4px indentation of the 'C-12' and 'C-13' text! This new bonus functionality really allows almost any type of styling possible, provided you have those CSS skillz.
+
+### The *combined* location helpers: `loc.column_header()`, `loc.footer()`, and `loc.header()`
+
+Look, I know we brought up the expression *fine-grained* before---right in the first paragraph---but sometimes you need just the opposite. There are lots of little locations in a GT table and some make for logical groupings. To that end, we have the concept of *combined* location helpers. There are three of these and they equate to some collection of location helpers:
+
+- `loc.header()` == `[loc.title(), loc.subtitle()]`
+- `loc.column_header()` == `[loc.column_labels(), loc.spanner_labels()]`
+- `loc.footer()` == `[loc.source_notes()]`
+
+That last one, `loc.footer()`, is currently just an alias for `loc.source_notes()`. However, in the future, when footnotes support is available in the package, the set will expand to include a `loc.footnotes()` location.
```

Comment:
It felt a bit confusing to see details on three combined selectors, but only two used in the example. It seems okay to mention the gist of combined selectors, with a tiny explanation after the example (can link to any relevant docs for a full enumeration)

```suggestion
Look, I know we brought up the expression *fine-grained* before---right in the first paragraph---but sometimes you need just the opposite. There are lots of little locations in a GT table and some make for logical groupings. To that end, we have the concept of *combined* location helpers.

```